Gopkg.toml

```diff
@@ -50,7 +50,7 @@
   name = "github.com/mailgun/metrics"
 
 [[constraint]]
-  revision = "fd192d755b00c968d312d23f521eb0cdc6f66bd0"
+  branch = "master"
```

Comment:
This is basically means that there is no constraint, so maybe it is better to remove constraints with `branch="master"` all together?

Reply:
Yup. Will do. Good point.